### ISSUE TITLE
create the map layout of the cemetery

### ISSUE THREAD

**Issue by alexitsios** [alexitsios/Calamity#104]
### Description

Create a map and relevant documentation of this location.

### Acceptance Criteria

The Map & Relevant documentation has been completed and placed into the wiki.
The Map & Relevant documentation follow the outline https://github.com/alexitsios/Calamity/wiki/Story-Bible#outline

**Comment by alexitsios** [comment]
@DimitrisKalyvas I checked your map layout, it looks good so far. You might want to add some items that the player finds, such as ammo, med kits etc. It's not clear to me if the player will be limited to walking only through the pathways to give more emphasis to closed corridors. Also, what's the purpose of the statue?

FYI @mashewmake @chaserimmer @FaceRadiation @ChristosDav @FreelanceSeven

**Comment by DimitrisKalyvas** [comment]
I also thought o adding items for the players to find but I'm not sure about that yet. Will look into it more. 

The path acts as a guide for players to see where they're going.

The statue is mentioned in the locations and acts as a minor landmark for the players.

**Comment by alexitsios** [comment]
moved this back to in progress till feedback has been provided.

**Comment by DimitrisKalyvas** [comment]
My main concerns with the cemetery is what purpose it serves basically. I don't see a reason for it to have any enemies as it's described as horror and spooky, so not adding enemies and using ambience is a good way to adhere to that. If we use large tomb-like structures for graves, and not small objects, we can create a maze basically, which would be a pretty good idea in my opinion, as it further disorientates the players and makes them scared. 

Regarding items, we can put them in dead-ends (I'd have to adjust some parts of the layout).

In (one of) the dead ends, we could have the puzzle piece(s).

**Comment by alexitsios** [comment]
@DimitrisKalyvas it's where they bury basically the bodies of the unworthy ones / experiments, so its a good enough reason for enemies to be present there. Etc the excuse could be that the parasites that escaped infected the dead bodies and some of them are lurking there, but not the advanced versions, but the basic ones.

**Comment by nicolai21** [comment]
There even could be partial or open graves that would allow the dead infected to be from.  Since the operations were interrupted it could explain those.  like Dimitris also says just having it where the threat of possible attacks could also put the player on edge.  That could be done with sounds, trees, or even sfx of shadows moving.

**Comment by nicolai21** [comment]
![Image](https://user-images.githubusercontent.com/9097391/223279831-f7f8cacd-6c09-4782-8b83-df1e99a37ee0.jpg)

here is an idea for a possible cemetery.  I looked at some older and monastery ones to get an idea.

**Comment by FreelanceSeven** [comment]
A bit basic, but that would make sense for a cemetery. Perhaps look at the outdoor areas of Spencer Mansion in RE1 Remake, for a bit more inspiration.

**Comment by nicolai21** [comment]
@FreelanceSeven 
![Cemetary Idea 2](https://user-images.githubusercontent.com/9097391/223574001-597a6f9d-87d1-479e-b107-42abdfb6d926.jpg)

![Cemetary Idea 3](https://user-images.githubusercontent.com/9097391/223575355-95275c1b-cfb8-4185-b747-9b595ec97a27.jpg)

Here are a couple new takes on it.  There could be some hill that could be in the graveyard to add depth to it.

**Comment by ChristosDav** [comment]
I really like the latest design sketch of @nicolai21 . Seems more full in general. 

I also like the combination of the mass graves. This could probably be something like large tomb structures, as @DimitrisKalyvas suggested (which is also a great idea). Makes the gameplay more tense.

Having some sort of a small slope would definitely add more depth to the location which is a plus.

As for the enemies and combat, I was thinking something like a chase between the Hunter and Cassandra in there. But we'll need to discuss it with Andrew, if this fits as a concept.

**Comment by DimitrisKalyvas** [comment]
I love the suggestions so far. I can only insist we try for a sort of maze feel for the cemetery to confuse and scare players, the atmosphere here would be perfect. If we add the chase concept as well, this will be an amazing horror experience.

**Comment by alexitsios** [comment]
@nicolai21 I like the organic feeling of the map. Well done! Can you create an iteration with a more maze-like feel as @DimitrisKalyvas suggests?

**Comment by nicolai21** [comment]
![Cemetary Idea 4](https://user-images.githubusercontent.com/9097391/223881260-75377ab8-8ff6-4354-bda7-393d167abd54.jpg)

I made it more mazelike with some walls inside as well.

**Comment by alexitsios** [comment]
I really like the new style @nicolai21 . @DimitrisKalyvas what do you think about it? Would something like this applicable for implementation?

**Comment by DimitrisKalyvas** [comment]
I absolutely love this. As a general feel, I think this is exactly what we want from this level in terms of what we want to achieve. My only concern right now is how large it is, but that is something we will have to test first before we can know for certain. I think we can implement this in the engine and we will take it one step at a time from there.

**Comment by mashewmake** [comment]
I really like this as well. I think the maze style will really freak out the players and will create potential for spooky interactions and there’s plenty of places where you can hide items. Make sure that you include where players start and even a player path so that it is easier to read.

**Comment by alexitsios** [comment]
cool, @nicolai21 can you replace current layout with your version here https://github.com/alexitsios/Calamity/wiki/Locations#cemetery ?

**Comment by alexitsios** [comment]
@nicolai21 I'm closing this and I'll set up a new task for you (blocking out the cemetery).

FYI @DimitrisKalyvas @FaceRadiation @mashewmake @chaserimmer @ChristosDav @FreelanceSeven

**Comment by alexitsios** [comment]
@mashewmake check out the following map and if you think it's good then you can mark it as completed.

Fyi @seekeroftheball

**Comment by mashewmake** [comment]
Map has been updated. This is now completed.

**Comment by crtherin** [comment]
@mashewmake can you update this to include map views?

**Comment by mashewmake** [comment]
Map has been updated with camera views and I have updated it in the wiki as well @crtherin
